package.json

```diff
@@ -7,43 +7,40 @@
     "test": "test"
   },
   "scripts": {
-    "start": "node dist/server.js",
```

Comment:
It may be usefull to specify with which engine this works and don't https://docs.npmjs.com/files/package.json#engines

Also maybe use the repo name as package.json name